corehq/apps/cloudcare/templates/formplayer/users.html

```diff
@@ -24,40 +24,44 @@ <h1 class="page-title">Login as user</h1>
       <tbody>
       </tbody>
     </table>
-    <nav class="module-pagination-container text-right">
-      <ul class="pagination">
-        <!-- Render Previous button -->
-        <% if (page === 1) { %>
-        <li class="disabled">
-          <% } else { %>
-        <li>
+    <% if (endPage) { %>
+      <% if (!(pageCount === 1 && limit === 10)) { %>
+        <div class="container">
+          <div class="row">
+            <div class="col-sm-3">
+              <div class="form-inline per-pagination-text">
+                <select class="form-control js-page-limit">
+                  <% for(var i = 0; i < rowRange.length; i++) { %>
+                    <% if (rowRange[i] === limit) { %>
+                      <option value="<%= rowRange[i] %>" selected>{% trans "<%= rowRange[i] %> per page" %}</option>
+                    <% } else { %>
+                      <option value="<%= rowRange[i] %>">{% trans "<%= rowRange[i] %> per page" %}</option>
+                    <% } %>
+                  <% } %>
+                </select>
+              </div>
+            </div>
+          <% if (pageCount > 1) { %>
+            <div class="col-sm-6">
```

Comment:
The alignment looks like it's working now regardless of screen size, which is great. One small tweak: could you add a `text-center` class here? That'll keep the pagination centered on wide screens, so it'll balance a bit better with the per page dropdown and the search box.